### PR TITLE
Cli docs

### PR DESCRIPTION
**Pull request by or-yam**
refernce from the redme file on greenpress-cli repo.
- change some of the styling to mach the docs site.
- removed the note about `greenpress start` command on windows.
- added note about `greenpress populate` command on node V15.

### CODE REVIEW

File: docs/tools/cli.md [hunk 1]
```diff
@@ -1,3 +1,140 @@
...
+
+usage: `greenpress h` or `greenpress --help`
+
+### **version**
+
+provides information about installed cli's version
+
+usage: `greenpress -V`, `greenpress --version`
+
+### **start**
```
Comment (drorrose) [inline docs/tools/cli.md:25]: debug is not supported anymore

File: docs/tools/cli.md [hunk 2]
```diff
@@ -1,3 +1,140 @@
...
+
+### **service**
+
+handles actions for easier development of services
+
+usage:
+
+`greenpress service [action][services]`
+
+supported services:
```
Comment (drorrose) [inline docs/tools/cli.md:109]: would move this line up to the top of the page (after installation, before describing commands), as it is crucial for both local option of start and for service create

File: docs/tools/cli.md [hunk 3]
```diff
@@ -1,3 +1,140 @@
...
+
+create custom themes.
+
+usage:
+
+`greenpress theme [options][name]`
+
+options:
+
+- `--from <theme>` set base theme to be used
```
Comment (drorrose) [inline docs/tools/cli.md:134]: --from=<theme>, wouldn't work without the equation mark

File: docs/tools/cli.md [hunk 4]
```diff
@@ -1,3 +1,140 @@
...
+    `greenpress start dev -l assets,front`
+
+    `greenpress start dev --local=assets,front`
+
+- exclude: choose which services not to start In production (input: `x`), the default will be to exclude the db service.<br>
+  E.g:
+
+  `greenpress start -x none`
+
+- debug: keeps the start process alive and display live application logs.
```
Comment (drorrose) [inline docs/tools/cli.md:49]: not supported anymore

**Review by drorrose — CHANGES_REQUESTED** [review]

File: docs/tools/cli.md (hunk 2 above)
Comment (or-yam) [reply docs/tools/cli.md:109]: did you mean to move  the  whole service comand section ?

File: docs/tools/cli.md [hunk 5]
```diff
@@ -8,6 +8,16 @@ A command-line interface to help you create and manage your Greenpress applicati
 npm install -g @greenpress/cli
 ```
 
+For commands that accepts a service_name as parameter, the supported services are:
+
+- auth, admin, assets, secrets, content, front
```
Comment (drorrose) [inline docs/tools/cli.md:13]: mailing is missing

File: docs/tools/cli.md [hunk 6]
```diff
@@ -8,6 +8,16 @@ A command-line interface to help you create and manage your Greenpress applicati
 npm install -g @greenpress/cli
 ```
 
+For commands that accepts a service_name as parameter, the supported services are:
```
Comment (drorrose) [inline docs/tools/cli.md:11]: give it a title, will be easier to read

File: docs/tools/cli.md [hunk 7]
```diff
@@ -137,4 +139,4 @@ options:
 
 E.g:
 
-`greenpress theme my-theme --from=classic`
+`greenpress theme my-theme --from classic`
```
Comment (drorrose) [inline docs/tools/cli.md:142]: why did you change it? it was correct

**Review by drorrose — CHANGES_REQUESTED** [review]

File: docs/tools/cli.md [hunk 8]
```diff
@@ -8,9 +8,11 @@ A command-line interface to help you create and manage your Greenpress applicati
 npm install -g @greenpress/cli
 ```
 
+## For using a specific services
```
Comment (drorrose) [inline docs/tools/cli.md:11]: a specific service/ using specific services

**Review by drorrose — CHANGES_REQUESTED** [review]

**Review by drorrose — APPROVED** [review]

**Review by davidmeirlevy — APPROVED** [review]

**Pull request merged**